app/javascript/App/bundles/NewsfeedPage/NewsfeedPage.tsx

```diff
@@ -206,11 +207,12 @@ class NewsfeedPage extends React.Component<IProps, IState> {
             </React.Fragment>
           }
           modalName="newsfeedModal"
+          // @ts-ignore
           modalSection={<BodySection />}
           drawerSection={
             <>
-              <CoinListDrawer />
-              <BodySectionDrawer bodySection={<BodySection />} />
+              {/* <CoinListDrawer />
+              <BodySectionDrawer bodySection={<BodySection />} /> */}
```

Comment:
Commenting out it for now. The mobile and tablet layout will be changed in another PR